src/main/resources/story/web/createBoardAndCards.story

```diff
@@ -0,0 +1,23 @@
+Scenario: Log-in
+When I login with ${email} and ${password}
+When I click on element located `id(login-submit)`
+
+Scenario: Create a new board
+When I wait until element located `By.xpath(//*[@data-testid="create-board-tile"])` appears
+When I click on element located `By.xpath(//*[@data-testid="create-board-tile"])`
+When I wait until element located `By.xpath(//*[@data-testid="create-board-title-input"])` appears
+When I enter `#{generate(regexify '[a-z]{5}[A-Z]{2}')}` in field located `By.xpath(//*[@data-testid="create-board-title-input"])`
+When I click on element located `By.xpath(//*[@data-testid="create-board-submit-button"])`
+When I wait until element located `By.xpath(//*[@data-testid="filter-popover-button"])` appears
+Then number of elements found by `By.xpath(//*[@data-testid="filter-popover-button"])` is = `1`
+
+Scenario: Create cards
+When I enter `<nameList>` in field located `cssSelector(.list-name-input)`
+When I click on element located `cssSelector(.mod-list-add-button)`
+When I click on element located `cssSelector(.js-open-card-composer)`
+When I enter `<nameCard>` in field located `cssSelector(.js-card-title)`
+When I click on element located `cssSelector(.js-add-card)`
```

Comment:
Same concern as for board scenario.
Please double check that created card is displayed on the website and that it has expected attribute values